coda-prototype/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "coda",
+  "version": "1.0.0",
+  "description": "Coda is an interactive tool that helps you label short text datasets!",
+  "main": "ui.js",
+  "dependencies": {
+    "karma": "^1.7.1",
```

Comment:
Why are these dependencies not dev dependencies?

Reply:
fixed